src/app/components/player/equipment/EquipmentPresets.tsx

```diff
@@ -92,6 +95,54 @@ const EquipmentPresets: React.FC = () => {
         };
         break;
       }
+      case EquipmentPreset.MID_LEVEL_MAGE: {
```

Comment:
I almost wonder if this should just be a set called Ahrim's? But then we would probably include Ahrim's staff which is uncommon

Reply:
Mid level mage is good. Otherwise people will be confused when there's "only" mid level melee and ranged.